programs/launcher/CMakeLists.txt

```diff
@@ -5,12 +5,31 @@ endif()
 
 find_package( Gperftools QUIET )
 if( GPERFTOOLS_FOUND )
-    message( STATUS "Found gperftools; compiling steemd with TCMalloc")
+    message( STATUS "Found gperftools; compiling with TCMalloc")
     list( APPEND PLATFORM_SPECIFIC_LIBS tcmalloc )
 endif()
 
-target_link_libraries( launcher
-                       PRIVATE fc ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/../../.git)
```

Comment:
Are these 3 cmake blocks the same?  Can we make a git.cmake file and use a macro/function?

Reply:
They are very similar, but they each set a different variable.  It's probably possible to make a cmake function type thing that accepts the variable name as a parameter, but I didn't take the time to figure out how.